pr_agent/tools/pr_reviewer.py

```diff
@@ -86,6 +88,9 @@ def _prepare_pr_review(self) -> str:
                 del data['PR Feedback']['Security concerns']
                 data['PR Analysis']['Security concerns'] = val
 
+        if settings.config.git_provider == 'github' and settings.pr_reviewer.inline_code_comments:
+            del data['PR Feedback']['Code suggestions']
```

Comment:
Consider adding a log message or a comment explaining why the 'Code suggestions' key is being deleted from the 'data' dictionary. This will improve code readability. [medium]